### ISSUE TITLE
ResNet50

### ISSUE THREAD

**Issue by superkirill** [themrzmaster/git-re-basin-pytorch#5]
Could you also provide a permutation spec for ResNet50?

**Comment by themrzmaster** [comment]
done it. let me know if it works as expected, my GPU takes a while to train a resnet50.
use depth 52 as an arg

**Comment by superkirill** [comment]
Thanks! My concern is that it's not a very "canonical" implemetation with 4 block groups and varying numbers of channels, so it's hard to use it with official PyTorch models

**Comment by themrzmaster** [comment]
Yes, it should be easy to use the same logic with resnet from torchvision. Just have to remember to not use batchnorm

**Comment by santosh9sanjeev** [comment]
Its not working with torchvisions ResNet50. Can you pls help me out
Thanks!

**Comment by adampan0527** [comment]
![cifar100_resnet52_4_weight_matching_interp_accuracy_epoch](https://github.com/user-attachments/assets/968f3900-749a-4e42-896e-858561bd502e)
Hi, whem I try ResNet50 on cifar100, the accuracy drops a lot, and permuted accuracy is even worse than naive which is unexpected.
